### ISSUE TITLE
questions about torchvision.transforms.Scale

### ISSUE THREAD

**Issue by icoz69** [pytorch/pytorch#2284]
hi,
i have questions when using torchvision.transforms.Scale to resize the training images
i want to resize all images to 32 * 128 pixels , what is the correct way ?

mine was :
transform = transforms.Compose(
[transforms.Scale((32,128)),
transforms.ToTensor(),
transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))])

but it turned out to be wrong. i change it to [32,128] or {32,128}, all wrong.

thanks

**Comment by apaszke** [comment]
Note that we're using GitHub issues for bug reports only, and all questions should be posted on [our forums](discuss.pytorch.org).
